python/lsst/ts/electrometer/controller.py

```diff
@@ -245,7 +255,7 @@ async def start_scan(self):
         await self.send_command(f"{self.commands.set_buffer_size(50000)}")
```

Comment:
I believe you can drop f"{..}", as factory method already returns string. Also you might want to change command_factory to return string directly:
```
    def activate_filter(self, mode, filter_type, active):
        """Return activate filter command.

        Parameters
        ----------
        mode : `UnitMode`
            The unit of the filter to activate.
        filter_type : `Filter`
            The filter type to activate
        active : `int`
            Whether to activate or not.

        Returns
        -------
        command : `str`
            The generated command string.
        """
        return f":sens:{enums.UnitMode(mode).name}:{enums.Filter(filter_type).name}:stat {int(active)};"
```